multimodal/src/autogluon/multimodal/configs/optimization/adamw.yaml

```diff
@@ -20,16 +21,21 @@ optimization:
   top_k_average_method: "greedy_soup"  # We support averaging method described in https://arxiv.org/pdf/2203.05482.pdf.
                                        # Currently support "uniform_soup", "greedy_soup", and "best".
   efficient_finetune:  # Can be 'bit_fit' (only finetune bias), 'norm_fit' (finetune the normalization terms + bias terms), lora (LoRA Adaptations only), lora_bias (LoRA Adaptation + bit_fit), lora_norm (LoRA Adaptation + norm_fit), or null
+  trainable_param_names: # If efficient_finetune is not None, specifies which layers should be trained.
+    - ".*head*."  # regex (re.match(trainable_param_name, name)
+    - ".*fusion_mlp*."
+    - ".*adapter*."
   lora:
-    r: 8
-    alpha: 8
-    filter: # Default fine-tune only query and value attention weights, recommended in https://arxiv.org/abs/2106.09685
+    adaptation_to_modules: # Specify which module (if any) to adapt(e.g. ".*EncDecAttention|.*DenseReluDense"). Default consider all modules in a model (i.e. empty filter).
+    adaptation_to_layers: # Specify which layer to adapt. Default fine-tune only query and value attention weights, recommended in https://arxiv.org/abs/2106.09685
```

Comment:
Can we have a reference to all the potential choices with explanation here and only leave default uncommented?

Reply:
Difficult to list them all as they depend on the specific architecture selected from the zoo of models available on Huggingface.